pkgs/games/steam/fhsenv.nix

```diff
@@ -264,83 +81,39 @@ in buildFHSEnv rec {
     #
     # [1] <https://github.com/libsdl-org/SDL/commit/8e2746cfb6e1f1a1da5088241a1440fd2535e321>
     export SDL_JOYSTICK_DISABLE_UDEV=1
-  '' + extraProfile;
 
-  runScript = writeShellScript "steam-wrapper.sh" ''
-    if [ -f /etc/NIXOS ]; then   # Check only useful on NixOS
-      ${mesa-demos-i686}/bin/glxinfo 2>&1 | grep -q Error
-      # If there was an error running glxinfo, we know something is wrong with the configuration
-      if [ $? -eq 0 ]; then
-        cat <<EOF > /dev/stderr
-    **
-    WARNING: Steam is not set up. Add the following options to /etc/nixos/configuration.nix
-    and then run \`sudo nixos-rebuild switch\`:
-    {
-      hardware.graphics.enable32Bit = true;
-      hardware.pulseaudio.support32Bit = true;
-    }
-    **
-    EOF
```

Comment:
We should at least move it into the description

Reply:
I don't think this check has actually ever worked properly tbh.

Reply:
it worked after I fixed it in https://github.com/NixOS/nixpkgs/commit/7f972523f363485afc6154f7ec195bc532316ce7